docs/basics/101-109-rerun.rst

```diff
@@ -0,0 +1,179 @@
+DataLad, Re-Run!
+----------------
+
+So far, you created a simple ``.tsv`` file of all
+speakers and talk titles in the longnow podcasts subdataset.
+Let's actually take a look into this file now:
+
+.. runrecord:: _examples/DL-101-108-109
+   :language: console
+   :workdir: dl-101/DataLad-101
+   :lines: 1-30
+
+   $ less recordings/Podcasts.tsv
+
+Not too bad, and certainly good enough for the podcast night people.
+What's been cool about creating this file is that it was created with
+a script within a ``datalad run`` command. Thanks to ``datalad run``,
+the output file ``Podcasts.tsv`` is associated with the script it
+generated.
+
+Upon reviewing the list you realized that you made a mistake, though: you only
+listed the talks in the SALT series (the
+``Long_Now__Seminars_About_Long_term_Thinking/`` directory), but not
+in the ``Long_Now__Conversations_at_The_Interval/`` directory.
+Lets fix this in the script. Replace the contents in ``code/list_titles.py``
+with the following, fixed script:
+
+.. runrecord:: _examples/DL-101-108-110
+   :language: console
+   :workdir: dl-101/DataLad-101
+   :emphasize-lines: 10
+
+   $ cat << EOT > code/list_titles.py
+   # import modules for path operations and pathname pattern expansion
+   from os.path import join, basename
+   from glob import glob
+
+   # define a path to the podcasts. Note that this is a relative path,
+   # pointing from the root of DataLad-101 to the podcasts:
+
+   path = join('recordings', 'longnow',
+               '*', '*')        # <-- here is the change!
```

Comment:
That is a lot of text to outline this small change. Maybe we keep the original and modified scripts as actual scripts in the repo, and just make sphinx show the diff between them here?

Reply:
Maybe I am missing something, but this also wouldn't enable a ``datalad rerun`` because the command would call a different file?

The ``emphasis-lines`` option is doing the best it can to highlight the change FWIW

Reply:
The commands/logic would not change at all, only what is shown. Will try this out later on...